### PR TITLE
Updating js build to include new possible number information.

### PR DESCRIPTION
**Pull request by lararennie**

### CODE REVIEW

File: javascript/i18n/phonenumbers/metadatafortesting.js [hunk 1]
```diff
@@ -65,52 +65,88 @@ i18n.phonenumbers.metadata.countryToMetadata = {
...
-,[,,"NA","NA"]
-,[,,"NA","NA"]
-,"AD",376,"00",,,,,,,1,,,[,,"NA","NA"]
-,,,[,,"NA","NA"]
-,[,,"NA","NA"]
-,,,[,,"NA","NA"]
-]
-,"AE":[,[,,"[1-9]\\d{8}","\\d{9}"]
+,[,,"NA","NA",,,,,,[-1]
+]
```
Comment (keghani) [inline javascript/i18n/phonenumbers/metadatafortesting.js]: Why don't we want the closing ] at the end of the previous line?

Comment (lararennie) [inline javascript/i18n/phonenumbers/metadatafortesting.js]: This is autogenerated. I don't know how it decides to line-break.

File: javascript/i18n/phonenumbers/phonemetadata.pb.js [hunk 2]
```diff
@@ -490,6 +504,130 @@ i18n.phonenumbers.PhoneNumberDesc.prototype.clearPossibleNumberPattern = functio
...
+  return /** @type {?number} */ (this.get$Value(9, index));
+};
+
+
+/**
+ * Gets the value of the possible_length field at the index given or the default value if not set.
+ * @param {number} index The index to lookup.
+ * @return {number} The value.
+ */
+i18n.phonenumbers.PhoneNumberDesc.prototype.getPossibleLengthOrDefault = function(index) {
```
Comment (lararennie) [inline javascript/i18n/phonenumbers/phonemetadata.pb.js]: To reduce diffs with auto-generated js file. This file is the same as what protoc would generate now, just checked in.

File: javascript/i18n/phonenumbers/phonemetadata.pb.js [hunk 3]
```diff
@@ -2185,23 +2338,23 @@ i18n.phonenumbers.NumberFormat.prototype.getDescriptor = function() {
         type: String
       }
     };
-    i18n.phonenumbers.NumberFormat.descriptor_ =
+    i18n.phonenumbers.NumberFormat.descriptor_ = descriptor =
```
Comment (lararennie) [inline javascript/i18n/phonenumbers/phonemetadata.pb.js]: I just copied from the generated file :) I think it's to make it less verbose at line 2192, it's just doing an alias.

**Pull request merged**